code/modules/antagonists/brother/brother.dm

```diff
@@ -96,6 +96,12 @@
 	message_admins("[key_name_admin(admin)] made [key_name_admin(new_owner)] and [key_name_admin(bro)] into blood brothers.")
 	log_admin("[key_name(admin)] made [key_name(new_owner)] and [key_name(bro)] into blood brothers.")
 
+/datum/antagonist/brother/check_mastery()
+	for(var/datum/mind/M in team.members)
+		if(M.current.stat == DEAD || M.current.z != 1)
```

Comment:
If they aren't dead and aren't on some arbritary z-level?
This should specifically check for is_station_level, or check if the antag completed the conditions for the escape objective.
This one seems very easy in comparison to the others.

Reply:
Brother is easier because it's incredibly rare (and also I couldn't think of anything better)